src/scanners/rdf.js

```diff
@@ -24,7 +24,7 @@ export default class RDFScanner extends BaseScanner {
         },
       }).parseFromString(this.contents, 'text/xml');
 
-      resolve(xmlDoc);
+      resolve(xmlDoc, null, {namespace: this.namespace});
```

Comment:
Wait, no, `_getContents` only returns the parsed contents to `scan`, so you shouldn't need to change this line at all.

```js
resolve(xmlDoc);
```
should be fine.